ouroboros-consensus/src/Ouroboros/Consensus/Node/ErrorPolicy.hs

```diff
@@ -27,8 +33,8 @@ import           Ouroboros.Consensus.Util.ResourceRegistry
                      (RegistryClosedException, ResourceRegistryThreadException,
                      TempRegistryException)
 
-consensusErrorPolicy :: ErrorPolicies
```

Comment:
I hadn't even considered this approach (i.e., not parameterising `ErrorPolicies` by `blk`), good idea! :+1: